msal-objc-articles/reference/Classes/class_m_s_a_l_account_enumeration_parameters.md

```diff
@@ -0,0 +1,53 @@
+---
+title: MSALAccountEnumerationParameters
+
+---
+
+# MSALAccountEnumerationParameters
+
+
+
+
+
+## Public Properties
+
+|                | Name           |
+| -------------- | -------------- |
+| BOOL | **[ignoreSignedInStatus](Classes/class_m_s_a_l_account_enumeration_parameters.md#property-ignoresignedinstatus)**  |
+| NSString * | **[identifier](Classes/class_m_s_a_l_account_enumeration_parameters.md#property-identifier)**  |
```

Comment:
Not sure if it's just an artifact of this being on a GitHub branch, but it seems like the hyperlinks all return `404` - maybe this is expected at this stage?